### PR TITLE
Added tests for REFing a parameter and having a failed test + passed test

### PR DESCRIPTION
**Pull request by Saarett**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by gruebel — APPROVED** [review]
🍰

File: tests/cloudformation/graph/checks/resources/LambdaFunction/template.yaml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 Description: X-ray tracing is enabled for Lambda.
+Parameters:
+  ActiveTracingConfig:
+    Description: Active tracing config
+    Type: String
+    Default: "Active"
+  WrongValueTracingConfig:
+    Description: Active tracing config
+    Type: String
+    Default: "Wrong value"
```
Comment (schosterbarak) [inline tests/cloudformation/graph/checks/resources/LambdaFunction/template.yaml:10]:
```suggestion
    Default: "PassThrough"
```

**Review by schosterbarak — APPROVED** [review]
Thanks

**Comment by schosterbarak** [comment]
@Saarett when committing my suggestion here: https://github.com/bridgecrewio/checkov/pull/2256/files/453593bacd247e40f03326b3330f6c33a40de019#r790320754 the test fails

**Pull request merged**